docs/design/limits_and_constraints/index.md

```diff
@@ -8,7 +8,9 @@ CARET only supports environments as follows.
 - Support FastDDS or CycloneDDS
 - Support only Linux, especially Ubuntu
 - Support Galactic and Humble
-- Require rebuilding of an application
+- Support only applications build with rclcpp library
+- Support only processes acting as node
```

Comment:
I don't think it is "Environment". maybe you can move this to "ROS 2 functions" or "Implementation", and description will be like "Other than processing that operates on a node. e.g. Launch"

Reply:
Thank you for your comment. I have fixed conditon type on [5373f84](https://github.com/tier4/CARET_doc/pull/202/commits/5373f8491a2430ab89a257d2500317c8a35008a2).